components/weather_data/WeatherStationDetail.tsx

```diff
@@ -32,8 +33,9 @@ interface columnData {
 }
 
 export const WeatherStationDetail: React.FC<Props> = ({center_id, stationId, source, requestedTime}) => {
+  const partnerCenterId = usePartnerCenterIdForStationFetch(center_id);
   const [days, setDays] = useState(1);
-  const avalancheCenterMetadataResult = useAvalancheCenterMetadata(center_id);
+  const avalancheCenterMetadataResult = useAvalancheCenterMetadata(partnerCenterId);
```

Comment:
This is only a partner center in the case of HPAC, so it's a bit misleading to name it as such - maybe keep the identifier and frame it as an override? 

```
const avalancheCenterMetadataResult = useAvalancheCenterMetadata(centerOrPartnerCenter(center_id));
```

We're just doing some logic (as opposed to using an API to drive the partner center designation) so we don't need to do a hook for it.